### DISCUSSION TITLE
how to use cypress-parallel with cucumber report

### DISCUSSION THREAD

**Discussion by peacetrader** [LironEr/cypress-mochawesome-reporter#190] [Q&A]
Hi, I am using cucumber with cypress-parallel. I am able to generate report cucumber.json using cypress-cucumber-preprocessor plugin. I specify the output in the package.json but the problem is when we run threads = 2 , the cucumber.json is generating for last thread execution, so the previous one is getting override. 
I can see documentation using mochawesome report only. Is there any help on how to use cucumber report for this. 
commands :
cypress-parallel -s cy:run -t 2 -a "cypress command args"
cy:run : "cypress run --browser edge"
currently in package.json we specify
"cypress-cucumber-preprocessor": {
"json": {
"enabled": true,
"output": "report/cucumber.json"
}
}

**Comment by LironEr** [comment]
I didnt try to use both parallel & cucumber

But you can try first make the cucumber part work, with this exmaple:
https://github.com/LironEr/cypress-mochawesome-reporter/tree/master/examples/cucumber

Then try to add parallel:
https://github.com/LironEr/cypress-mochawesome-reporter/tree/master/examples/cypress-parallel

Let me know if you need more help
